drivers/SmartThings/matter-lock/profiles/lock-without-codes-or-tamper.yml

```diff
@@ -0,0 +1,21 @@
+name: lock-without-codes-or-tamper
```

Comment:
I would suggest changing this to `lock-nocodes-notamper` to follow the naming precedent of excluding capabilities that is used in the `matter-thermostat` profiles: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/tree/main/drivers/SmartThings/matter-thermostat/profiles

When changing the name, remember to change the `name:` field, filename, and the profile name in the fingerprint above.